src/RawCommand.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef CONTROLDEV_RawCommand_HPP
```

Comment:
Why mixed case - would have expected all upper case here?

Reply:
Does it matter ?

It's mixed case because it was easier to implement as a snippet in vscode.

Reply:
I thought that is convention, isn't it?  Plus adding (yet) another header guard style to this library.

Reply:
> I thought that is convention, isn't it?

It is. Now, I thought that breaking the case convention (not the actual define name) for the guards wouldn't be such a big deal, given how so much better it is to have a template to generate them. But I'll fix it anyways.

Reply:
Fixed

Reply:
I use `${TM_FILENAME_BASE/(.*)/${1:/upcase}/}_${TM_FILENAME/^.*\\.([^\\.]*)$/${1:/upcase}/}` in my snippet

Reply:
To be perfectly compliant with "the convention", it would also need to do CamelCase to UPPER_SNAKE_CASE. In any case, just updated the snippet to do the uppercase, if you feel lucky and manage to have a magical sentence to do the camelcase, I'd be happy to know about it too ;-)

Reply:
`${TM_FILENAME_BASE/([a-z]*)(([A-Z])+([a-z]+))?/${1:/upcase}${2:+_}$3${4:/upcase}/g}` ?

:-P